kuma/attachments/management/commands/configure_social_auth.py

```diff
@@ -0,0 +1,88 @@
+import fileinput
```

Comment:
Why is this script in the `kuma/attachments` app? Perhaps a mistake?

Reply:
Moved! Thanks for catching that mistake